bin/node/runtime/src/lib.rs

```diff
@@ -268,8 +269,8 @@ impl pallet_session::Config for Runtime {
 }
 
 impl pallet_session::historical::Config for Runtime {
-    type FullIdentification = ();
-    type FullIdentificationOf = ();
+    type FullIdentification = pallet_staking::Exposure<AccountId, Balance>;
+	type FullIdentificationOf = pallet_staking::ExposureOf<Runtime>;
```

Comment:
I see some inconsistent indentation. Try running `cargo fmt --all` for code formatting.

(ref: https://github.com/rust-lang/rustfmt#quick-start)

Reply:
Thanks @hoonsubin,
formatting done